### ISSUE TITLE
bug: [alt+tab]ing away from full-scrn windows triggers notification

### ISSUE THREAD

**Issue by jzacsh** [eonpatapon/gnome-shell-extension-caffeine#63]
**tl;dr** I'm currently seeing notifications when I don't expect them. Does the behavior I describe below sound intended? If not, is it this extension's fault or does this sound like a gnome bug?

From the [extension's listener](https://github.com/eonpatapon/gnome-shell-extension-caffeine/blob/bf0ec29ffb1a6/caffeine%40patapon.info/extension.js#L228-L229) and [its binding](https://github.com/eonpatapon/gnome-shell-extension-caffeine/blob/bf0ec29ffb1a6/caffeine%40patapon.info/extension.js#L107-L108), it seems the only explanation is a buggy gnome-shell is [emitting `org.gnome.SessionManager::InhibitorRemoved`](https://people.gnome.org/~mccann/gnome-session/docs/gnome-session.html#org.gnome.SessionManager::InhibitorRemoved) when it shouldn't?

---

All the specifics I can think of:
```bash
$ gnome-shell --version
GNOME Shell 3.18.1

$ lsb_release --all
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux testing-updates (sid)
Release:	testing-updates
Codename:	sid

$ uname -a
Linux carboy 4.3.5 #1 SMP Sun Feb 28 20:09:54 EST 2016 x86_64 GNU/Linux
```

- Steps to reproduce
 1. open browser
 2. open new window in browser and fullscreen that window
 3. wait 2 seconds
 4. type `alt`+`~` to switch between browser windows
- Expected behavior:

  No new notifications should occur _(since steps 1-3 don't involved interacting with the extension)_
- Actual behavior

  step 3 produces a notification "Auto suspend and screensaver enabled"

**Comment by jzacsh** [comment]
btw, is there a more appropriate/canonical place to see that API documentation? (the link I gave was fom a sad looking set of search results against `InhibitorRemoved`)

**Comment by eonpatapon** [comment]
You must have the auto-enable feature on for full screen windows.

So when you go fullscreen the extension activates. There is a notification but you don't see it. When you alt-tab to some other window there is no full screen window so the extension deactivates hence the notification.

**Comment by jzacsh** [comment]
Hmm.. I can't find a place where such preferences exist. Can you point me in the right direction?

Tangentially, could I simply patch this extension to not enable when already enabled? It seems like that's the problem?

**Comment by eonpatapon** [comment]
Just look in the extension preferences (gnome-shell-extension-prefs)

**Comment by jzacsh** [comment]
oh wow, I don't know where one learns about that, but that's great! Thanks so much :)
